.browserslistrc

```diff
@@ -0,0 +1,4 @@
+# Browser support for the DarkReader API
+last 2 major versions
```

Comment:
Hmm `last 2 major versions`, our DarkReader API should support ES5 as it's gets transpiled to ES5
https://github.com/darkreader/darkreader/blob/master/tasks/bundle-api.js#L27 not sure how this can be interpenetrated in the browserlist syntax.

Reply:
@Gusted I made sure that the browser supports `es5` and `css-filters` in 5d8da73.